apps/src/studio/Item.js

```diff
@@ -113,6 +300,25 @@ Item.prototype.removeElement = function () {
   Studio.trackedBehavior.removedItemCount++;
 };
 
+
+/**
+ * Returns true if the item is currently fading away.
+ */
+Item.prototype.isFading = function() {
+  return this.startFadeTime ? true : false;
+};
+
+
+/**
+ * Returns true if the item has finished fading away.  The caller will usually
+ * then call removeElement to destroy this item's assets.
+ */
+Item.prototype.isGone = function() {
```

Comment:
might consider a more clear name like `hasCompletedFade` or something to that effect...